include/deal.II/physics/transformations.h

```diff
@@ -711,26 +759,27 @@ namespace Physics
 //@{
 
     /**
-    * Returns the result of applying Nanson's formula for the transformation of
-    * the material surface area element $d\mathbf{A}$ to the current surfaces
-    * area element $d\mathbf{a}$ under the nonlinear transformation map
-    * $\mathbf{x} = \boldsymbol{\varphi} \left( \mathbf{X} \right)$.
-    *
-    * The returned result is the spatial normal scaled by the ratio of areas
-    * between the reference and spatial surface elements, i.e.
-    * @f[
-    *  \mathbf{n} \frac{da}{dA}
-    *    := \textrm{det} \mathbf{F} \, \mathbf{F}^{-T} \cdot \mathbf{N}
-    *     = \textrm{cof} \mathbf{F} \cdot \mathbf{N} \, .
-    * @f]
-    *
-    * @param[in] N The referential normal unit vector $\mathbf{N}$
-    * @param[in] F The deformation gradient tensor $\mathbf{F} \left( \mathbf{X} \right)$
-    * @return      The scaled spatial normal vector $\mathbf{n} \frac{da}{dA}$
-    *
-    * @dealiiHolzapfelA{75,2.55}
-    * @dealiiWriggersA{23,3.11}
-    */
+     * Returns the result of applying Nanson's formula for the transformation
+     * of the material surface area element $d\mathbf{A}$ to the current
+     * surfaces area element $d\mathbf{a}$ under the nonlinear transformation
+     * map $\mathbf{x} = \boldsymbol{\varphi} \left( \mathbf{X} \right)$.
+     *
+     * The returned result is the spatial normal scaled by the ratio of areas
+     * between the reference and spatial surface elements, i.e.
+     * @f[
+     *  \mathbf{n} \frac{da}{dA}
+     *    := \textrm{det} \mathbf{F} \, \mathbf{F}^{-T} \cdot \mathbf{N}
+     *     = \textrm{cof} \mathbf{F} \cdot \mathbf{N} \, .
+     * @f]
+     *
+     * @param[in] N The referential normal unit vector $\mathbf{N}$
+     * @param[in] F The deformation gradient tensor $\mathbf{F} \left(
+     * \mathbf{X} \right)$
+     * @return      The scaled spatial normal vector $\mathbf{n}
+     * \frac{da}{dA}$
+     *
+     * @dealiiHolzapfelA{75,2.55} @dealiiWriggersA{23,3.11}
```

Comment:
I don't know how this is going to render, but I'm not going to worry about it right now.